ui/print.go

```diff
@@ -42,8 +41,10 @@ func Fatal(msg string, err error) {
 		}
 		uie := &Error{Error: e}
 		fmt.Println(uie.Verbose())
-		os.Exit(1)
+	} else {
+		fmt.Println(errorColor.Render(wordwrap.String(msg, MaxOutputLineLength)))
 	}
+	os.Exit(1)
```

Comment:
This is not right. The API for this is meant to always display the `msg` and additionally display the error. It should always os.Exit though

Reply:
Oh I see, sorry for the misunderstanding